lib/parser/builders/default.rb

```diff
@@ -1803,17 +1803,17 @@ def module_definition_map(keyword_t, name_e, operator_t, end_t)
     end
 
     def definition_map(keyword_t, operator_t, name_t, end_t)
-      Source::Map::Definition.new(loc(keyword_t),
-                                  loc(operator_t), loc(name_t),
-                                  loc(end_t))
+      Source::Map::MethodDefinition.new(loc(keyword_t),
+                                        loc(operator_t), loc(name_t),
+                                        loc(end_t), nil, nil)
     end
 
     def endless_definition_map(keyword_t, operator_t, name_t, assignment_t, body_e)
```

Comment:
let's rename this method to `method_definition_map`

Reply:
Sure. But we'll need to keep an alias for compatibility, right?

Reply:
`endless_definition_map` is (was) used only in `ruby28.y` that is still in development. I think it's ok to rename without any aliases

Reply:
Oh, sorry, I thought you wanted `definition_map` => `method_definition_map`.

Would you like to rename `endless_definition_map` => `method_definition_map` and that this is used for standard and endless method definitions?

My PR uses `definition_map` for standard methods, and `endless_definition_map` for endless ones. This way `definition_map` remains as a valid API and the implementation is updated, but maybe there's a better way to go.

Reply:
(seems it would be confusing if standard methods used `definition_map` and endless ones used `method_definition_map`)

Reply:
nvm, I think it's better to keep it as is.

EDIT: just got a notification from GH that you reverted it. Yes, let's keep it as is, otherwise it sounds like it handles all method definitions. 👍

Reply:
Yes, I had misunderstood your comment (although you clearly made it on the `def endless_definition_map`, I should have paid more attention)